### ISSUE TITLE
Improve Interactive Highlights in BASE

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#516]
Interactive highlights were enabled in BASE in #514. They work pretty well but @terracoda requested an improvement to make the highlight look dashed when the balloons are grabbed, like they behave with keyboard interaction.

This won't be possible until we implement the general support for having a custom Interactive Highlight, which was also requested in https://github.com/phetsims/friction/issues/200. We will work on this in scenery. Once that is done, the work for this issue can continue in the sim.

But the change is not of critical importance and is low priority at the moment, upcoming publications can proceed without this.

**Comment by jessegreenberg** [comment]
Support for this was added to scenery, this can be done now.

**Comment by jbphet** [comment]
@emily-phet, @Ashton-Morris, and I tried this out on master during today's sound design meeting and it seems like it is working the way it is supposed to, meaning that the interactive highlights look the same when using the mouse and when using keyboard navigation to move the balloons.  Is there anything else that needs to be done on this?

**Comment by jessegreenberg** [comment]
No, this has been resolved by common code changes. @terracoda was the one to originally request this, FYI this has been done.
